Samples/PublicSamples/EchoBot/src/EchoBot/Bot/BotMediaStream.cs

```diff
@@ -163,8 +142,6 @@ private async Task StartAudioVideoFramePlayerAsync()
         {
             try
             {
-                await Task.WhenAll(this.audioSendStatusActive.Task).ConfigureAwait(false);
```

Comment:
Do we not need to wait on this task anymore ?

Reply:
I found this had race condition issues between creating the audio video player and starting to process the audio before the player was finished being set up. I don't think we need the media status to be active before we create the audio player. Before we start processing the stream of data we check to make sure the player has finished being set up.